app/helpers/sessions.rb

```diff
@@ -0,0 +1,24 @@
+helpers do
+
+  def current_user
+    if session[:volunteer_id]
+      @current_user ||= Volunteer.find(session[:volunteer_id])
+    end
+  end
+
+  def logged_in?
+    !current_user.nil?
+  end
+
+  def current_admin
```

Comment:
`current_admin` should be removed. You should either be signed in as an admin user or not. 

```ruby
helpers do
  def logged_in_as_admin?
    current_user && current_user.admin?
  end
end
```